src/x509_str.c

```diff
@@ -1850,6 +1850,13 @@ WOLF_STACK_OF(WOLFSSL_X509_OBJECT)* wolfSSL_X509_STORE_get0_objects(
 #if defined(WOLFSSL_SIGNER_DER_CERT) && !defined(NO_FILESYSTEM)
     cert_stack = wolfSSL_CertManagerGetCerts(store->cm);
```

Comment:
Why would we get `NULL` here? Isn't `NULL` supposed to be an error return? The way I'm reading `wolfSSL_CertManagerGetCerts`, it can return an empty stack but a `NULL` return should signify an error occurred.

Reply:
In wolfssl_cm_get_certs_der() if numCerts is zero then it causes an error. I thought the current fix I made was easier than rewriting the internals of wolfssl_cm_get_certs_der() to allow zero certs.